src/controller.coffee

```diff
@@ -1202,11 +1202,18 @@ define ['droplet-helper',
             # We're at the end of a non-empty line.
```

Comment:
This comment may need to be updated since it doesn't seem like the indentation is handled in this clause anymore. It isn't clear to me from reading the new code how leadingWhiteSpace is added to the first line (since it is skipped if index==0 down below)